examples/vae/README.md

```diff
@@ -0,0 +1,17 @@
+# Basic VAE Example
+
+This is an implementation of the paper [Auto-Encoding with Variational Bayes](http://arxiv.org/abs/1312.6114) by D.P.Kingma and M.Welling.
+This code follows [pytorch/examples/vae](https://github.com/pytorch/examples/blob/master/vae/README.md).
+
+```bash
+pip install -r requirements.txt
+python main.py
+```
+
+## Examples;
+
+If you run the code by above command, you can get some generated images:
+
+![generated_mnist](https://github.com/makora9143/flax/blob/examples/vae/examples/vae/example.png)
```

Comment:
please use relative links see: https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-readmes#relative-links-and-image-paths-in-readme-files